src/components/NearbyShops.vue

```diff
@@ -26,9 +19,9 @@ export default
             return this.$store.getters.nearbyShops;
         },
     },
-    methods:
+    components:
     {
-
+        ShopList,
```

Comment:
I thought ShopList is explicit enough. It renders a list of shops. Should be called ShopListView?
Should the name indicate somehow that we're dealing with a dumb component? What would you suggest? ;)